lib/elixir/lib/system.ex

```diff
@@ -73,11 +104,27 @@ defmodule System do
   @doc """
   Elixir build information.
 
-  Returns a keyword list with Elixir version, git short revision hash and compilation date.
+  Returns a keyword list with Elixir version, compilation date, git tag and short revision hash.
   """
   @spec build_info() :: map
   def build_info do
-    %{version: version, date: get_date, revision: get_revision}
+    %{date:     get_date,
+      revision: binary_part(get_revision, 0, 7),
+      tag:      get_tag,
+      version:  version,
+      }
+  end
+
+  @doc false
+  # Returns a string of the build info
+  @spec build_info(atom :: :version_build) :: String.t
```

Comment:
Let's get rid of this exported function please. We need to find a way to include this in the `build_info` otherwise we need to cut it out.